bootstrap.sh

```diff
@@ -43,22 +52,15 @@ else
   echo -e "WARNING: Please make sure this configuration works in this vagrant box!\n\n" 
 fi
 
-# Configure the app if it isn't
-if [ ! -f /vagrant/src/api/config/options.yml ] && [ -f /vagrant/src/api/config/options.yml.example ]; then
-  echo "Configuring your app in config/options.yml..." 
-  sed 's/source_port: 5352/source_port: 3200/' /vagrant/src/api/config/options.yml.example > /vagrant/src/api/config/options.yml
-else
-  echo -e "\n\nWARNING: You have already configured your app in config/options.yml." 
-  echo -e "WARNING: Please make sure this configuration works in this vagrant box!\n\n" 
-fi 
 
 echo "Setting up your OBS test backend..."
 # Put the backend data dir outside the shared folder so it can use hardlinks
 # which isn't possible with VirtualBox shared folders...
 mkdir /tmp/vagrant_tmp
 chown vagrant:users /tmp/vagrant_tmp
+chown vagrant:users -R /vagrant/src/api/log/*
```

Comment:
Sorry, took me some time to understand as I`m not using virtualbox (I use libvirt) and vagrant-libvirt does not use share folders, its rsyncing to the box and files owned by root afterwards. 

I will modify this to use an "bind" filesystem like you do with tmp. Would this be a solution for you ?